### DISCUSSION TITLE
Different Wallets: same clientid and same verifier

### DISCUSSION THREAD

**Discussion by andrebruch** [Web3Auth/Web3Auth#566] [Help]
A user got a different wallet despite having same clientId, same aggregateVerifier (tkey-auth0-email-passwordless), same verifier (torus) and same verifierId.
He logged in on 6/13/22 and 8/02/22.

The app has been running in production for a few months and the problem has only occurred with one account so far.

**Comment by YZhenY** [comment]
Could we get more details on the users account? Whats his email and is there logs? I'll pm you on tele, im zenzhenyu.

**Reply by shahbaz17** [reply]
Making this done, hoping this is moved to telegram.
